server/src/index.js

```diff
@@ -1,13 +1,18 @@
 const express = require('express');
+const { getRestaurantReviews } = require('../../database/src/index.js');
 
 const PORT = process.env.PORT || 3000;
 
 const app = express();
 app.use(express.static('public'));
 app.use(express.json());
 
-app.get('/hello', (req, res) => {
-  res.status(200).json(req);
+app.get('/restaurants/:restaurantId/reviews', (req, res) => {
```

Comment:
Interesting, will this handle different IDs thrown at it?

Reply:
This will handle different IDs - the `:propName` syntax makes that prop available as a variable on express' req.params object. In this case, I destructure the req.params object to define the restaurantId variable locally on line 11, and then pass it into the getRestaurantReviews function.

However, I did not do any type checking - I'll add some type checking to ensure it's a number, and let the user know if they didn't pass a number!

Reply:
One of the seniors told me about this syntax but I have yet to use it.  Great thinking!